sentry-android-okhttp/src/main/java/io/sentry/android/okhttp/SentryOkHttpInterceptor.kt

```diff
@@ -40,7 +41,9 @@ class SentryOkHttpInterceptor(
                 span.toSentryTrace().let {
                     requestBuilder.addHeader(it.name, it.value)
                 }
-                span.toBaggageHeader()?.let {
+
+                span.toBaggageHeader(request.headers(BaggageHeader.BAGGAGE_HEADER))?.let {
+                    requestBuilder.removeHeader(BaggageHeader.BAGGAGE_HEADER)
```

Comment:
This removes the current `header` and adds it again, but they should be merged, or does `span.toBaggageHeader` merges them internally?

Reply:
Apparently `toBaggageHeader` merges them, ignore it.